### PR TITLE
add sample registries.conf to docs

### LINKED ISSUE #1385: Need canonical registries.conf
I'm creating a script to automatically install Buildah and a number of other tools on Ubuntu, but I need a `/etc/containers/registries.conf`. There's a [sample in tests](https://github.com/containers/buildah/blob/master/tests/registries.conf) and also references to [the Fedora version](https://src.fedoraproject.org/rpms/skopeo/blob/master/f/registries.conf), but it would be nice to have an official one, or have a suggested option documented. For now I am just doing:

    curl -sL https://raw.githubusercontent.com/containers/buildah/master/tests/registries.conf | \
          sudo tee /etc/containers/registries.conf

But it would feel more reliable if the directory was `sample/config` rather than `tests`.

### PR DESCRIPTION
**Pull request by QiWang19**
fix #1385 
Signed-off-by: Qi Wang <qiwan@redhat.com>

### CODE REVIEW

File: docs/samples/registries.conf [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# It adheres to TOML format and does not support recursive
+# lists of registries.
+
+# The default location for this configuration file is /etc/containers/registries.conf.
+
+# The only valid categories are: 'registries.search', 'registries.insecure',
+# and 'registries.block'.
+
+[registries.search]
+registries = ['docker.io', 'quay.io']
```
Comment (TomSweeneyRedHat) [inline docs/samples/registries.conf]: I'd flip the order, put quay first, docker second

**Comment by TomSweeneyRedHat** [comment]
One tiny nit, otherwise LGTM

**Comment by rhatdan** [comment]
SGTM

File: docs/samples/registries.conf [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+[registries.insecure]
+registries = []
+
+
+# If you need to block pull access from a registry, uncomment the section below
+# and add the registries fully-qualified name.
+#
+# Docker only
+[registries.block]
+registries = []
```
Comment (rhatdan) [inline docs/samples/registries.conf]: Add newline.

Comment (QiWang19) [reply docs/samples/registries.conf]: fixed :raising_hand_woman:

File: docs/samples/registries.conf (hunk 1 above)
Comment (QiWang19) [reply docs/samples/registries.conf]: fixed :raising_hand_woman:

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by TomSweeneyRedHat** [comment]
LGTM, nice job @QiWang19 !

**Pull request closed without merging**